build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 group = 'com.iexec.core'
 
 ext {
-    springCloudVersion = '2021.0.0'
+    springCloudVersion = '2021.0.5'
```

Comment:
I'm not sure the spring-cloud dependencies are still required here.

Reply:
TODO added in https://github.com/iExecBlockchainComputing/iexec-core/pull/542/commits/0c7279239ef805abec49008bb705459af6a28b8a. Let's remove it in a later PR :)